server/controllers/courses/course.js

```diff
@@ -0,0 +1,25 @@
+const getCourseInformation = require('../../database/queries/getCourse');
+
+exports.getAllCourses = (req, res) => {
```

Comment:
@AbdallahAmmar96 Should this be named `getCourse` instead?  I think it's designed to only return one course, not all of them.